pkgs/applications/video/gpu-screen-recorder/default.nix

```
@@ -22,14 +25,15 @@
 
 stdenv.mkDerivation {
   pname = "gpu-screen-recorder";
-  version = "unstable-2024-07-05";
+  version = "unstable-2024-09-03";
```

```suggestion
  version = "4.1.11";
```

https://github.com/NixOS/nixpkgs/pull/339874#issuecomment-2395240334

Looks like tagged releases are now recommended.